project_task_default_stage/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,6 @@
+* Carlos Dauden <carlos@incaser.es>
+* Sergio Teruel <sergio@incaser.es>
+* Duc, Dao Dong <duc.dd@komit-consulting.com> (https://komit-consulting.com)
+* `Tecnativa <https://www.tecnativa.com>`_:
+
+  * Ernesto Tejeda
```

Comment:
Add contributors in the following format:
Ernesto Tejeda youremail@tecnativa.com (Tecnativa https://www.tecnativa.com)